internal/luasandbox/apis.go

```diff
@@ -114,3 +116,208 @@ func (l *LuaEnvironment) apiStrSplit() API {
 		`,
 	}
 }
+
+func (l *LuaEnvironment) apiGitReadBlob() API {
+	return &GoAPI{
+		Name:      "gitReadBlob",
+		Signature: "gitReadBlob(hash) -> blob bytes",
+		Help:      "Retrieve the Git blob in byte form from the repository with the provided hash.",
+		Examples: []string{
+			"gitReadBlob(abcdef) -> 'Hello, world!'",
+		},
+		Implementation: func(s *lua.LState) int {
+			object := s.ToString(1)
+			hash, err := gitinterface.NewHash(object)
+			if err != nil {
+				s.Push(lua.LString(err.Error()))
+				return 1
+			}
+
+			blob, err := l.repository.ReadBlob(hash)
+			if err != nil {
+				s.Push(lua.LString(err.Error()))
+				return 1
+			}
+			s.Push(lua.LString(blob))
+			return 0
```

Comment:
shouldn't this be return 1? am I misunderstanding this? I assumed it was "pop n from stack"